README.md

```diff
@@ -27,11 +27,13 @@ const foo = {#
 <details><summary>Click here to see the desugared version</summary>
 
 ```js
-const foo = Object.freeze({
-  a: Object.freeze({
-    b: Object.freeze({
-      c: Object.freeze({
-        d: Object.freeze({
+const freeze = obj => Object.freeze(Object.create(null), Object.assign(o, obj));
```

Comment:
I think this should be `Object.freeze(Object.assign(Object.create(null), object))`

Reply:
woops